platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/admin/core/api/ConfigurationProperties.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.core.api;
+
+import java.util.Map;
+
+/**
+ * Properties of a configuration
```

Comment:
✏️  Can you add more JavaDoc to `ConfigurationProperties ` and `ConfigurationDetails`?  I still do not understand what the difference is between the two.  Currently I would need to go back and see how they are both used to understand the difference between details and properties.